docs/dev/design_considerations.rst

```diff
@@ -94,51 +94,77 @@ At time of writing, the repository includes the following subpackages:
 * truelearn/models: contains the definitions of event model and learner model.
 * truelearn/preprocessing: contains the preprocessing utilities, such as Wikifier.
 * truelearn/tests: contains all the tests for TrueLearn library.
-* truelearn/utils: contains two utility packages, ``metrics`` (contains scoring functions) and ``visualisations``.
+* truelearn/utils: contains two utility packages, ``metrics`` (contains scoring functions) and ``visualisations`` (contains a variety of visualizations
+  that can be used to visualize the learner's knowledge).
```

Comment:
visualisations, use a different word instead of visualise since its a lot of repetition